apps/backend/src/poll/poll-actions.service.ts

```diff
@@ -190,6 +196,11 @@ export class PollActionsService implements OnModuleInit {
     return this.pollModel.findById(id).exec();
   }
 
+  // Only for internal use
+  async findParticipant(id: Types.ObjectId): Promise<Doc<Participant> | null> {
```

Comment:
Please remove and use PollService.find

Reply:
I've used `participantService.find()` instead